README.md

```diff
@@ -1,20 +1,21 @@
 # repytah
 
-A Python package for building Aligned Hierarchies for music-based data streams.
```

Comment:
Just my random idea but will it make sense to combine these two, so something like "sequential music-based data streams"? I checked against the thesis and while I believe "sequential data" is the correct term to use, since this is the beginning of the README file perhaps it will be nice if we can go a bit more specific, ofc as long as it makes sense though.

Reply:
I feel like we can ask KK if she wants the readme to be bonded with music and then decide whether to put "music-based". I'm fine with these two!

Reply:
I agree with Zoie. The point is if we want music as a keyword or we want to go beyond the MIR area.

Reply:
Correct me if I am wrong, but I remember Katie telling us that this package was meant to be used with any data that has repeated structures, and musical data is just one type. @zoiezhao we talked about this on Thursday, right, when we stayed behind after the meeting?

Reply:
anyhow, we can wait to publish the package, till we know for sure :)